Makefile

```diff
@@ -164,12 +162,14 @@ $(DASHBOARD_SERVER) -d \
 @$(MAKE) test-container \
 	CMD=$(CMD) \
 	SERVER_NAME=$(IT_SERVER_NAME) \
-	DOCKER_OPTIONS="-e APP_URL=http://$(IT_IP):$(HTTP_PORT) \
-		-e KERNEL_GATEWAY_URL=http://$(IT_IP):$(IT_KG_PORT) \
+	DOCKER_OPTIONS="-e APP_URL=http://$(DASHBOARD_CONTAINER_NAME):$(HTTP_PORT) \
+		-e KERNEL_GATEWAY_URL=http://$(KG_CONTAINER_NAME):$(IT_KG_PORT) \
```

Comment:
`IT_KG_PORT` is still referenced in 2 places, but not defined. In other parts of the Makefile, we explicitly use `8888` with `KG_CONTAINER_NAME`. Probably best to pull that out into a `KG_PORT` var.